docs/reference/sql/endpoints/client-apps/index.asciidoc

```diff
@@ -5,19 +5,33 @@
 
 beta[]
 
-Thanks to its <<sql-jdbc, JDBC>> interface, a broad range of third-party applications can use {es}'s SQL capabilities.
+Thanks to its <<sql-jdbc, JDBC>> and <<sql-odbc, ODBC>> interfaces, a broad range of third-party applications can use {es}'s SQL capabilities.
 This section lists, in alphabetical order, a number of them and their respective configuration - the list however is by no means comprehensive (feel free to https://www.elastic.co/blog/art-of-pull-request[submit a PR] to improve it):
 as long as the app can use the {es-sql} driver, it can use {es-sql}.
 
 * <<sql-client-apps-dbeaver, DBeaver>>
 * <<sql-client-apps-dbvis, DbVisualizer>>
+* <<sql-odbc-applications-excel, Microsoft Excel>>

```

Comment:
Please update the section label as well from `sql-odbc-application-XXX` to `sql-client-apps-XXX`.
I don't think having odbc in the header is useful either for us or the user.